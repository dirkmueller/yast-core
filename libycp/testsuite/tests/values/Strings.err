Parsed:
----------------------------------------------------------------------
"Hello world"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"Hello          world"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"Hello world$"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"Hello world$"
----------------------------------------------------------------------
tests/values/Strings.ycp:24:bad octal constant
[Parser] tests/values/Strings.ycp:24 bad octal constant
Parsed:
----------------------------------------------------------------------
"Hello world"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"Hello\r\n\t\b\fworldHello"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"Special characters: !#$%&\264()*+,-./:;<=>?@[]^_`{}|~"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"Hello world\n Hello world"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"Hello world Hello world"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"Hello world999Hallo world"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"Hello world\0044Hallo world"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"Hello world$Hallo world"
----------------------------------------------------------------------
