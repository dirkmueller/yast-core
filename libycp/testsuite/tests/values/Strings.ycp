
# ---------------------------------------------------------
#
#  Filename:    Strings.ycp
#
#  Purpose:     test cases for YCP test
#
#  Creator:     jd@suse.de
#
#  Maintainer:  jd@suse.de
#
# ---------------------------------------------------------


// this tests includes tests for YCP_String

# ------- YCP_String
"Hello world"
"Hello \
         world"     // breaking strings into more lines

"Hello world\44"
"Hello world\044"
"Hello world\755"	// octal const too large
"Hello\r\n\t\b\fworld\
Hello"
"Special characters: !#$%&´()*+,-./:;<=>?@[]^_`{}|~"
"Hello world
 Hello world"
"Hello world\
 Hello world"

"Hello world\999Hallo world"
"Hello world\0044Hallo world"
"Hello world\044Hallo world"
