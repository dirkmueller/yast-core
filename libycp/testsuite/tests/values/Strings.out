("Hello world")
("Hello          world")
("Hello world$")
("Hello world$")
("Hello world")
("Hello\r\n\t\b\fworldHello")
("Special characters: !#$%&\264()*+,-./:;<=>?@[]^_`{}|~")
("Hello world\n Hello world")
("Hello world Hello world")
("Hello world999Hallo world")
("Hello world\0044Hallo world")
("Hello world$Hallo world")
